commons/all/src/main/java/org/infinispan/commons/configuration/attributes/AttributeDefinition.java

```diff
@@ -28,52 +27,41 @@
  */
 public final class AttributeDefinition<T> {
    private final String name;
-   private final String xmlName;
    private final T defaultValue;
    private final boolean immutable;
    private final boolean autoPersist;
    private final boolean global;
    private final AttributeCopier copier;
```

Comment:
nit
missing type `<T>` in `copier` field and methods.

Reply:
Yes